modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/ding_ddbasic_opening_hours.theme.inc

```diff
@@ -273,3 +273,171 @@ function template_preprocess_ding_ddbasic_opening_hours_all(&$variables) {
     $structured
   );
 }
+
+/**
+ * Categorized library opening hours.
+ */
+function theme_ding_ddbasic_opening_hours_all_tabbed($variables) {
+  return render($variables['tabbed']);
```

Comment:
Based on the `template_preprocess_ding_ddbasic_opening_hours_all_tabbed` variable `$variables['tabbed']` is already rendered in line 442 (see https://github.com/easySuite/ding2/pull/344/files#diff-9cf518f5f8d70d2545eb3f003295b475R442). I suppose that only one rendering is necessary and one render function call can be removed.